app/managers/performance_metrics_manager/types.d.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+type PerformanceReportMeasure = {
+    metric: string;
+    value: number;
+    timestamp: number;
+}
+
+type PerformanceReport = {
```

Comment:
Regarding my other comment about `@mattermost/types`, I should move these types to that package as well. I forgot that mobile would need them too

Reply:
Same consideration about server versioning, but 0/5